code/__DEFINES/subsystems.dm

```diff
@@ -334,3 +334,10 @@
 #define VOTE_COUNT_METHOD_SINGLE 1
 /// Approval voting. Any number of selections per person, and the selection with the most votes wins.
 #define VOTE_COUNT_METHOD_MULTI 2
+
+/// The choice with the most votes wins. Ties are broken by the first choice to reach that number of votes.
+#define VOTE_WINNER_METHOD_SIMPLE 1
```

Comment:
These should be strings unless there's a reason for them to be numbers--it's harder to VV these